pdd.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "1.5",
+    "description": "Date/time difference calculator and countdown timer. The name 'pdd' stands for Python Date Diff.",
+    "homepage": "https://github.com/jarun/pdd",
+    "license": "GPL-3.0-or-later",
+    "notes": [
+        "pdd requires dateutil to run properly",
+        "You can install it via pip: pip install python-dateutil"
+    ],
+    "suggest": {
+        "Python": "python"
+    },
+    "url": "https://github.com/jarun/pdd/raw/v1.5/pdd#/pdd.py",
```

```suggestion
    "url": "https://raw.githubusercontent.com/jarun/pdd/v1.5/pdd#/pdd.py",
```

Better to avoid redirection